### ISSUE TITLE
Support for specified sample request, response and state objects

### ISSUE THREAD

**Issue by adrianffletcher** [accordproject/cicero-template-library#339]
It would be great to be able to specify a sample request, response, state and event object rather than just randomly generating data for the documentation of a Smart Clause Template.

**Describe the solution you'd like**
I would like to be able to specify a sample json file for the request, response and state objects in the package.json file which would be used when generating the documentation for a Smart Clause Template. I would like this to default to request.json, response.json, state.json and event.json if not specified.

Example of current Helloworld sample:
<img width="792" alt="Screenshot 2020-04-15 at 13 51 16" src="https://user-images.githubusercontent.com/29235349/79339093-40bc8880-7f20-11ea-954f-343f8f827b0b.png">

**Comment by jeromesimeon** [comment]
I don't think we check that those `requests.json` `state.json` etc in the template archives are valid and work at the moment. This would be a prerequisite to switch the samples in the template library documentation to those present in the template. See related issue https://github.com/accordproject/cicero-template-library/issues/76

**Comment by adrianffletcher** [comment]
Doesn't cicero trigger by definition check that the request and state are valid?

**Comment by jeromesimeon** [comment]
Moving this to Cicero, where I believe the corresponding work would need to happen.
